frappe/public/js/frappe/auto_repeat_schedule.html

```diff
@@ -0,0 +1,20 @@
+<table class="table table-bordered small">
```

Comment:
If you move this file to the `auto_repeat` doctype folder, this will be added along with the doctype, no need to add it in the build! This will reduce bloat in the general build too.

Hopefully this is the last fix. Will be good to merge after this.